src/legacy/core_plugins/console/public/src/controllers/sense_controller.js

```diff
@@ -35,10 +40,16 @@ module.run(function ($rootScope) {
   };
 });
 
+function showWelcomeMessageIfNeeded() {
+  if (true || storage.get('version_welcome_shown') !== '@@SENSE_REVISION') {
```

Comment:
I think `true` here is leftover from testing? Because now the welcome flyout is always shown when I navigate to Console. 😄

Reply:
oops!